src/web/app/pages-instructor/instructor-session-individual-extension-page/individual-extension-date-modal/individual-extension-date-modal.component.ts

```diff
@@ -154,21 +154,21 @@ export class IndividualExtensionDateModalComponent {
   }
 
   isCustomizeDateTimeIntegers(): boolean {
-    if (this.isCustomize()) {
-      if (!Number.isInteger(this.extendByDatePicker.days) || !Number.isInteger(this.extendByDatePicker.hours)) {
-        return false;
-      }
+    if (!this.isCustomize()) {
```

Comment:
`isCustomize` => `isCustomized`?

Reply:
I agree, though it actually means is "customize"... well can't think of a more appropriate name for now